[2025-01-29T09:08:57.473+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-29T09:08:57.496+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my-first-python-etl-dag.extract manual__2025-01-29T09:08:53.304238+00:00 [queued]>
[2025-01-29T09:08:57.511+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my-first-python-etl-dag.extract manual__2025-01-29T09:08:53.304238+00:00 [queued]>
[2025-01-29T09:08:57.512+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-29T09:08:57.545+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract> on 2025-01-29 09:08:53.304238+00:00
[2025-01-29T09:08:57.555+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=500) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-29T09:08:57.559+0000] {standard_task_runner.py:72} INFO - Started process 502 to run task
[2025-01-29T09:08:57.558+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my-first-python-etl-dag', 'extract', 'manual__2025-01-29T09:08:53.304238+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dag.py', '--cfg-path', '/tmp/tmpv1k9nlxm']
[2025-01-29T09:08:57.561+0000] {standard_task_runner.py:105} INFO - Job 15: Subtask extract
[2025-01-29T09:08:57.917+0000] {task_command.py:467} INFO - Running <TaskInstance: my-first-python-etl-dag.extract manual__2025-01-29T09:08:53.304238+00:00 [running]> on host 0d287206d23f
[2025-01-29T09:08:58.014+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='olalekanrasaq1331@gmail.com' AIRFLOW_CTX_DAG_OWNER='Akinkunmi' AIRFLOW_CTX_DAG_ID='my-first-python-etl-dag' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2025-01-29T09:08:53.304238+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-29T09:08:53.304238+00:00'
[2025-01-29T09:08:58.015+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-29T09:08:58.016+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-29T09:08:58.017+0000] {logging_mixin.py:190} INFO - Current task name:extract state:running start_date:2025-01-29 09:08:57.497713+00:00
[2025-01-29T09:08:58.017+0000] {logging_mixin.py:190} INFO - Dag name:my-first-python-etl-dag and current dag run status:running
[2025-01-29T09:08:58.018+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-29T09:08:58.019+0000] {logging_mixin.py:190} INFO - Inside Extract
[2025-01-29T09:08:58.019+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-29T09:08:58.031+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-29T09:08:58.032+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=my-first-python-etl-dag, task_id=extract, run_id=manual__2025-01-29T09:08:53.304238+00:00, execution_date=20250129T090853, start_date=20250129T090857, end_date=20250129T090858
[2025-01-29T09:08:58.046+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-29T09:08:58.047+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-29T09:08:58.047+0000] {logging_mixin.py:190} INFO - Dag name:my-first-python-etl-dag queued_at:2025-01-29 09:08:53.318988+00:00
[2025-01-29T09:08:58.048+0000] {logging_mixin.py:190} INFO - Task hostname:0d287206d23f operator:PythonOperator
[2025-01-29T09:08:58.098+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-29T09:08:58.304+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-29T09:08:58.307+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
